docs/docs/troubleshooting.md

```diff
@@ -59,3 +59,16 @@ After opening the `<board>.dts.pre.tmp:<line number>` and scrolling down to the
 |  ![Healthy Keymap Temp](../docs/assets/troubleshooting/keymaps/healthyEDIT.png)  |
 | :-------------------------------------------------------------------------------: |
 |  A properly defined keymap with successful compilation. As shown in red, the corrected keycode (`&kp SPC`) references the proper Usage ID defined in the [USB HID Usage Tables](https://www.usb.org/document-library/hid-usage-tables-12)|
+
+### Split Keyboard Halves Unable to Pair
+
+The previous method of pairing split keyboard halves involved a **BLE Reset** with a specific combination of held keys that would remove all bluetooth profile information from the keyboard.
+Since then, a much simpler procedure of performing a bluetooth reset for split keyboards has come about, without the need for any file modification:
+
+1. Log into Github and download the "settings clear" UF2 image from latest GH actions from the `main` of ZMK
```

Comment:
Let's at least link to the GH Actions for the main repo from here, to make it easier to get there directly.

Reply:
@petejohanson 
Which of the following links would be most appropriate? I'd imagine the first out of this list:
https://github.com/zmkfirmware/zmk/actions
https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild
https://github.com/zmkfirmware/zmk/actions?query=workflow%3ATest
https://github.com/zmkfirmware/zmk/actions?query=workflow%3Aclang-format-lint

Or should I link that step to a specific workflow within actions?

Reply:
We can go one step better, and specifically get the `Build` job(s) only from `main`, if we use https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain